### PR TITLE
Fix broken refcard merge

### PR DESCRIPTION
**Pull request by jjaderberg**
- Fix incomplete forward merge in https://github.com/neo4j/neo4j/commit/0ffad41cef810001719bb20d287bd719507c25b4
- Incorporate feedback that came in after PR was merged.

### CODE REVIEW

File: manual/cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/PatternsTest.scala [hunk 1]
```diff
@@ -176,22 +172,18 @@ WHERE id(n) = %A% AND id(m) = %B%
 
 RETURN n, m###
 
-Veriable lenth path of any number of relationships from `n` to `m`.
+Variable lenth path of any number of relationships from `n` to `m`.
```
Comment (Mats-SX) [inline manual/cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/PatternsTest.scala:175]: `length`

File: manual/cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/PatternsTest.scala [hunk 2]
```diff
@@ -176,22 +172,18 @@ WHERE id(n) = %A% AND id(m) = %B%
...
 MATCH (n) WHERE id(n) = %A%
->>>>>>> upstream/2.3
 CREATE UNIQUE
 
 (n)-[:KNOWS]->(m {property: {value}})
 
 RETURN m###
 
-A relationship of type `KNOWS` from `n` to an `m` that has the declared property.
+A relationship of type `KNOWS` from `n` to an `m` having the declared property.
```
Comment (Mats-SX) [inline manual/cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/PatternsTest.scala:186]: I suggest `from a node n to a node m with the ...`

File: manual/refcard/src/cypher-refcard.asciidoc [hunk 3]
```diff
@@ -20,7 +20,7 @@ or read the full Cypher documentation in the
 For live graph models using Cypher check out <a href="http://gist.neo4j.org" target="_blank">GraphGist</a>.</p></div>
 ++++
 
-The Cypher Refcard is also link:{docs-home}/pdf/neo4j-cypher-refcard-{neo4j-version}.pdf[available as PDF].
+The Cypher Refcard is also link:{docs-home}/pdf/neo4j-cypher-refcard-{neo4j-version}.pdf[available as a PDF].
```
Comment (Mats-SX) [inline manual/refcard/src/cypher-refcard.asciidoc:23]: How about `available in PDF format`?

**Comment by systay** [comment]
@Mats-SX good feedback. we'll have to take care of it in a separate PR. want to get the builds green ASAP

**Pull request merged**